src/Traits/ManagesItemsTrait.php

```diff
@@ -393,4 +396,82 @@ protected function performProtectedCheck($item)
             throw new ModifyingProtectedValueException("Cannot access $item because it is protected");
         }
     }
+
+    /**
+     * Hydrate with external data
+     *
+     * @param $type  string    The type of data to be hydrated into the manager
+     * @param $data string     The data to be hydrated into the manager
+     * @return $this
+     * @throws \Michaels\Manager\Exceptions\SerializationTypeNotSupportedException
+     */
+    public function hydrateFrom($type, $data)
+    {
+        // we can possibly do some polymorphism for any other serialization types later
+        if($type !== 'json'){
```

Comment:
Let's `strtolower()` and `trim()` type in case someone passes in `'  JsOn ` for whatever stupid reason.